concent_api/test_server_script.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+import sys
+import json
+from json import JSONDecodeError
+import datetime
+import random
+
+import requests
+
+
+DEFAULT_HEADERS = {
+    'Content-Type': 'application/json',
+    'concent-client-public-key': '85cZzVjahnRpUBwm0zlNnqTdYom1LF1P1WNShLg17cmhN2UssnPrCjHKTi5susO3wrr/q07eswumbL82b4HgOw==',
+}
+
+
+class InvalidOption(Exception):
+    pass
+
+
+def create_data(message_type):
+    current_time = int(datetime.datetime.now().timestamp())
+    data = {
+        "type":         message_type,
+        "timestamp":    current_time,
+        "message_task_to_compute": {
+            "type":         "MessageTaskToCompute",
+            "timestamp":    current_time,
+            "task_id":      5,
+            "deadline":     current_time + 6000
+        }
+    }
+    if message_type == "MessageRejectReportComputedTask":
+        data = {
+            "type":         "MessageRejectReportComputedTask",
+            "timestamp":    current_time,
+            "reason":       "cannot-compute-task",
+            "message_cannot_commpute_task": {
+                "type":         "MessageCannotComputeTask",
+                "timestamp":    current_time,
+                "reason":       "provider-quit",
+                "task_id":      5
+            }
+        }
+    return data
+
+
+def api_request(host, endpoint, task_id = None, data = None, headers = None):
+    if headers is None:
+        headers = DEFAULT_HEADERS
+
+    if data is None:
+        print('Send message {}:'.format(endpoint))
```

Comment:
Wouldn't it be clearer to print `Receive message:` instead of `Send message receive` here? `receive` is not a valid message type. It's the name of an endpoint.